easybuild/easyconfigs/f/FastEpistasis/FastEpistasis-2.07-foss-2018a.eb

```diff
@@ -0,0 +1,39 @@
+#
+# This file is an EasyBuild reciPY as per github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2021 University of Geneva Switzerland
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+#
+easyblock = 'CMakeMake'
+
+name = 'FastEpistasis'
+
+version = '2.07'
+
+homepage = 'https://gitlab.sib.swiss/tschuepb/FastEpistasis'
+
+description = '''
+   FastEpistasis, a software tool capable of computing tests of epistasis for a large number of SNP pairs, is an 
+   efficient parallel extension to the PLINK epistasis module.'''
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://gitlab.sib.swiss/tschuepb/FastEpistasis/uploads/864695cfa08720b46dd1b437faca3fff/']
+sources = ['%(name)s-%(version)s.tar.bz2']
+checksums = ['3751461a0c410749ce77b07b957f7e9c4cbfb18aa4003b5aab3fc52db10b5f33']
+
+dependencies = [
```

Comment:
@ysagon You should change this to `builddependencies`.

`CMake` is only needed at build time, it's not a runtime dependency.

The test fails because the `cget` easyconfig file has the same issue, and we have a check to avoid introducing multiple "dependency variants" in a particular generation of easyconfigs (but build-only dependencies are ignored there)

Reply:
Hi, interesting to know there is a check to avoid dependency variants.

I modified the eb.